pkg/objectstore/objectstore_test.go

```diff
@@ -471,6 +471,26 @@ func (s *ObjectStoreProviderSuite) TestBucketGetRegions(c *C) {
 	c.Assert(err, IsNil)
 }
 
+func (s *ObjectStoreProviderSuite) TestGCSRegions(c *C) {
+	if s.osType != ProviderTypeGCS {
+		c.Skip("Test only applicable to GCS")
+	}
+	// GCS returns in UPPERCASE and checkv1.Equals is case sensitive
+	gcsRegions := []string{"US-WEST1", "EUROPE-WEST8", "ASIA-SOUTH2"}
+	ctx := context.Background()
+	for _, region := range gcsRegions {
+		s.initProvider(c, region)
+		bucketName := s.createBucketName(c)
+		_, err := s.provider.CreateBucket(ctx, bucketName)
+		c.Check(err, IsNil)
+		b, getErr := s.provider.GetBucket(ctx, bucketName)
+		err = s.provider.DeleteBucket(ctx, bucketName)
+		c.Check(getErr, IsNil)
```

Comment:
Might wanna check `getErr` before attempting to delete the bucket.

Reply:
But then in the rare chance that create passes & get fails then you don't attempt bucket cleanup. Defer is prohibited or not encouraged inside a for loop

Reply:
I see what you mean. You can use `defer` inside a go routine or closure within the `for` loop. It's just a take-it-or-leave-it suggestion.